includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -167,6 +169,11 @@ class AMP_Style_Sanitizer extends AMP_Base_Sanitizer {
 		'skip_tree_shaking'              => false,
 		'allow_excessive_css'            => false,
 		'transform_important_qualifiers' => true,
+		'font_face_display_overrides'    => [
+			'NonBreakingSpaceOverride' => 'optional',
+			'Inter var'                => 'optional',
+			'Genericons'               => 'auto',
```

Comment:
After considering this more, I think `block` is the better value to explicitly say we don't want the font to render anything until it loads.

```suggestion
			'Genericons'               => 'block',
```

Reply:
From what I read, browsers treat `auto` as “[often similar](https://css-tricks.com/almanac/properties/f/font-display/)” to `block`, but there seems some uncertainty.